quic/s2n-quic-tls/Cargo.toml

```diff
@@ -19,8 +19,8 @@ bytes = { version = "1", default-features = false }
 errno = "0.3"
 libc = "0.2"
 s2n-codec = { version = "=0.6.1", path = "../../common/s2n-codec", default-features = false }
```

Comment:
Looks like `s2n-codec` had some changes since the last release, so that should get a bump too